serverless_llm/store/serverless_llm_store/transformers.py

```diff
@@ -255,7 +259,8 @@ def best_effort_load(
         logger.debug(f"device_map: {device_map}")
     # check if 'cpu' is in device_map values and raise an exception
     if "cpu" in device_map.values():
-        raise ValueError("CPU is not supported in device_map.")
+        raise ValueError("GPU unavailable or not enough GPU memory.")
```

Comment:
Could you pls polish the error report message with ChatGPT?

Reply:
Changed to "The GPUs are either unavailable or lack sufficient memory. Please make sure they are ready."

Reply:
Why is "particularly the first GPU"?

Reply:
That's my experience, but actually, the error will be raised when GPU memory is not enough for a larger model with idle GPU0 and occupied GPU1. I have deleted this conditional sentence.

Reply:
I think this version is good, just delete "which may be due to external tasks using them, particularly the first GPU in your cluster".

Reply:
Done